app/routes/_layout+/bookings.$bookingId.add-assets.tsx

```diff
@@ -138,6 +139,28 @@ export async function loader({ context, request, params }: LoaderFunctionArgs) {
       userOrganizations,
       request,
     });
+
+    /** Self service can only manage assets for bookings that are DRAFT */
+    const cantManageAssetsAsBase =
```

Comment:
Is this something we discussed. I dont see it being part of the issue. It makes sense in a way but I need to have it a bit more clear what is included in the issue.

Reply:
I saw this check was only on frontend in file `booking-assets-column.tsx` line `57`. So I thought it is better to also implement the check on backend.